awxkit/awxkit/cli/format.py

```diff
@@ -20,16 +36,20 @@ def add_authentication_arguments(parser, env):
         help='an OAuth2.0 token (get one by using `awx login`)',
         metavar='TEXT',
     )
+
+    config_username, config_password = get_config_credentials()
+    # options configured via cli args take higher precedence than those from the config
     auth.add_argument(
         '--conf.username',
-        default=env.get('CONTROLLER_USERNAME', env.get('TOWER_USERNAME', 'admin')),
+        default=env.get('CONTROLLER_USERNAME', env.get('TOWER_USERNAME', config_username)),
         metavar='TEXT',
     )
     auth.add_argument(
         '--conf.password',
-        default=env.get('CONTROLLER_PASSWORD', env.get('TOWER_PASSWORD', 'password')),
+        default=env.get('CONTROLLER_PASSWORD', env.get('TOWER_PASSWORD', config_password)),
```

Comment:
the default won't show in the help text, will it?

(sorry for the surface-level question, I'm just starting to go over this)

Reply:
no, not as long `argparse.ArgumentDefaultsHelpFormatter` is not used

![image](https://user-images.githubusercontent.com/28510156/135733653-4491ac4f-ff8c-4baf-9c07-ddf6b52cf1b2.png)